spec/factories/legal_aid_applications.rb

```diff
@@ -32,13 +32,15 @@
         state_machine = FactoryBot.build(:non_passported_state_machine, legal_aid_application: application)
         application.update!(state_machine: state_machine)
       end
+      non_passported
```

Comment:
This is to set benefit_check_result to the right thing, however for some reason when the factory builds on application using `:non_passported_state_machine` or `:passported_state_machine` it doesn't update it, meaning `legal_aid_application.non_passported?` could be inaccurate.